packages/apps/job-launcher/server/src/modules/auth/auth.service.ts

```diff
@@ -114,11 +118,11 @@ export class AuthService {
   }
 
   public async logout(user: UserEntity): Promise<void> {
-    await this.authRepository.delete({ userId: user.id });
+    await this.authRepository.deleteByUserId(user.id);
```

Comment:
We can remove this service method since it doesn't have any logic apart from deleting tokens from DB.

Reply:
how would you do it then?

Reply:
I think what @Dzeranov means is to move `await this.authRepository.deleteByUserId(user.id);` directly to the methods that call the logout service method. @portuu3 @flopez7